packages/melange/melange.1.0.0/opam

```diff
@@ -15,7 +15,7 @@ depends: [
   "ounit" {with-test}
   "reason" {with-test}
   "ppxlib"
```

Comment:
This might be a fix:

```suggestion
  "ppxlib" {>= "0.24.0"}
```

It seems that Ast_414 is supported only [from ppxlib.0.24.0](https://toxis.caelum.ci.dev/github/ocaml/opam-repository/commit/b80e57a42a74c3a40a462867c9b700676729c086/variant/macos,macos-homebrew-ocaml-4.14-amd64,petrol.1.2.3,tests)

(Not sure whether menhir.20201214 is compatible with it though..)

Reply:
Thanks, this should work indeed. I'l return the fix upstream.